src/components/Testimonials/Testimonials.js

```diff
@@ -1,55 +1,63 @@
 // @flow
 
 import React from 'react'
+import PropTypes from 'prop-types'
 import withStyles from 'isomorphic-style-loader/lib/withStyles'
 import Slider from 'react-slick'
 
+import TestimonialItems from './TestimonialItems'
 import s from './Testimonials.css'
 
 type Props = {
-  testimonials?: Array<string>,
+  testimonials?: Array<Object>,
 }
 
 const Testimonials = ({ testimonials }: Props) => {
+  const countSlides = testimonials.length > 3
   const settings = {
     dots: false,
     infinite: false,
     speed: 500,
-    slidesToShow: 3.5,
+    slidesToShow: 3.3,
     slidesToScroll: 1,
+    focusOnSelect: true,
+    draggable: false,
+    arrows: countSlides,
```

Comment:
put `testimonials.length > 3` directly here, no need to declare it above, it's obvious what it is